src/screens/StarterPack/StarterPackScreen.tsx

```diff
@@ -327,42 +328,52 @@ function Header({
 
     setIsProcessing(true)
 
+    let listItems: AppBskyGraphDefs.ListItemView[] = []
     try {
-      const list = await agent.app.bsky.graph.getList({
-        list: starterPack.list.uri,
-      })
-      const dids = list.data.items
-        .filter(
-          li =>
-            li.subject.did !== currentAccount?.did &&
-            !isBlockedOrBlocking(li.subject) &&
-            !isMuted(li.subject) &&
-            !li.subject.viewer?.following,
-        )
-        .map(li => li.subject.did)
-
-      const followUris = await bulkWriteFollows(agent, dids)
-
-      batchedUpdates(() => {
-        for (let did of dids) {
-          updateProfileShadow(queryClient, did, {
-            followingUri: followUris.get(did),
-          })
-        }
+      listItems = await getAllListMembers(agent, starterPack.list.uri)
+    } catch (e) {
+      setIsProcessing(false)
+      Toast.show(_(msg`An error occurred while trying to follow all`), 'xmark')
+      logger.error('Failed to get list members for starter pack', {
```

Comment:
This toast seems to refer to a different error? 🤔

Reply:
This is all part of the same `Follow all` action for the user. We just want to log a different error message to Sentry.

Reply:
Oops, sorry

Reply:
all good, appreciate you looking over it!! ❤️